packages/assets-controllers/src/TokenDetectionController.ts

```diff
@@ -186,19 +188,19 @@ export class TokenDetectionController extends StaticIntervalPollingController<
    * @param options.messenger - The controller messaging system.
    * @param options.disabled - If set to true, all network requests are blocked.
    * @param options.interval - Polling interval used to fetch new token rates
-   * @param options.selectedAddress - Vault selected address
+   * @param options.selectedAccountId - Vault selected address
    * @param options.getBalancesInSingleCall - Gets the balances of a list of tokens for the given address.
    * @param options.trackMetaMetricsEvent - Sets options for MetaMetrics event tracking.
    */
   constructor({
-    selectedAddress,
+    selectedAccountId,
```

Comment:
Similar comment as [here](https://github.com/MetaMask/core/pull/4244#discussion_r1626310421) — what are your thoughts on using the messenger to get the current value of this later on in the constructor rather than having the controller take this as an argument?